### PR TITLE
Feature eslint style formatter

### PR DESCRIPTION
**Pull request by Exelord**
Hi!
I've just redesigned the formatter to look like Eslint. It also shows the count of all errors. Now fixing and debugging is even more simple are more readable.

@rwjblue What do you think about it?
screen shot 2017-07-15 at 16 13 52

### CODE REVIEW

File: bin/ember-template-lint.js [hunk 1]
```diff
@@ -7,14 +7,27 @@ var path = require('path');
...
   } else {
+    let count = 0;
+
     Object.keys(errors).forEach(filePath => {
-      console.log(Linter.errorsToMessages(errors[filePath]));
+      let options = {};
+      let fileErrors = errors[filePath] || [];
+      count += fileErrors.length;
+
+      if (process.argv.includes('--desc')) {
```
Comment (rwjblue) [inline bin/ember-template-lint.js]: Can we tweak the flag? Perhaps `--verbose`?

**Comment by Exelord** [comment]
@rwjblue Thank you :) I applied the feedback but I can't figure out why the tests are failing under node 4 :/

**Comment by Exelord** [comment]
Do you think about integrating this plugin with ESLint itself?

**Comment by rwjblue** [comment]
> Do you think about integrating this plugin with ESLint itself?

No, I haven't really considered that.

File: bin/ember-template-lint.js [hunk 2]
```diff
@@ -7,14 +7,27 @@ var path = require('path');
...
   } else {
+    let count = 0;
+
     Object.keys(errors).forEach(filePath => {
-      console.log(Linter.errorsToMessages(errors[filePath]));
+      let options = {};
+      let fileErrors = errors[filePath] || [];
+      count += fileErrors.length;
+
+      if (process.argv.includes('--verbose')) {
```
Comment (rwjblue) [inline bin/ember-template-lint.js]: Per http://node.green/#ES2016-features-Array-prototype-includes-Array-prototype-includes, Node 4 does not have a `Array.prototype.includes` method.

**Comment by Exelord** [comment]
My bad :/ Thank you for pointing out :) Fixed.

**Comment by rwjblue** [comment]
Thank you!!!

**Pull request merged**